lib/pushmi_pullyu/cli.rb

```diff
@@ -89,6 +95,20 @@ def parse_options(argv)
         config.monitor = true
       end
 
+      opts.on('-rh', '--redis-host IP', 'Host IP of Redis instance to read from.'\
```

Comment:
I like what you have here. I guess in the example above, you would want the 'config' directory to be bundled as part of the gem? (If you don't do that, where does it get it's default config file location from?)